scripts/control_device.py

```diff
@@ -213,7 +213,7 @@
     choices=possible_states.keys(),
     required=True,
     help=f"thermostat state, possible values: {possible_states}",
-    default=None,
+    default="",
```

Comment:
why do we need a default value for a required argument?

Reply:
because it is required for the control_thermostat function.

The argparse module won't assign value for this argument if its not required and no default value specified

Reply:
my question was why do we need a default value (based on it being required), not why is it required...

a "required" marked argument is an argument that is required as input by the user,
the argparse module will break the parsing process if it's missing.

an argument that has a "default" value is, by logic, not a "required" argument,
if the user doesn't input it, the argparse module uses the default value for it.

using both "required" and "default" makes no sense, please remove one.
unless, of course, I'm missing something, if so, please try explaining why again.